articles/container-service/container-service-monitoring-dynatrace.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Monitor an Azure Container Service cluster with Dynatrace | Microsoft Docs
+description: Monitor an Azure Container Service cluster with Dynatrace.
+services: container-service
+documentationcenter: ''
+author: MartinGoodwell
+manager: ''
+editor: ''
+tags: acs, azure-container-service
+keywords: Containers, DC/OS, Azure
+
+ms.assetid: 91d9a28a-3a52-4194-879e-30f2fa3d946b
+ms.service: container-service
+ms.devlang: na
+ms.topic: get-started-article
+ms.tgt_pltfrm: na
+ms.workload: na
+ms.date: 08/08/2016
+ms.author: t-ribhat
+
+---
+# Monitor an Azure Container Service cluster with Dynatrace SaaS/Managed
+In this article, we will deploy the Dynatrace OneAgent to all the agent nodes in your Azure Container Service cluster. You need an account with Dynatrace SaaS/Managed for this configuration. 
+
+## Prerequisites 
+[Deploy](container-service-deployment.md) and [connect](container-service-connect.md) a cluster configured by Azure Container Service. Explore the [Marathon UI](container-service-mesos-marathon-ui.md). Go to [https://www.dynatrace.com/trial/](https://www.dynatrace.com/trial/) to set up a Dynatrace SaaS account.  
+
+## Dynatrace SaaS/Managed
+Dynatrace is cloud-native monitoring for highly dynamic container and cluster environments. Dynatrace allows you to better optimize your container deployments and memory allocations by using real-time usage data. Dynatrace is the only solution capable of automatically pinpointing application and infrastructure issues in seconds using artificial intelligence.
```

Comment:
Consider moving this to the intro, people want to know what the article covers early on.